financials/Funding request template.txt

```diff
@@ -0,0 +1,24 @@
+## Template for funding request
+This funding request template was made based on past experiences and debates on the Grin Forum about the necessity to have clear, 
+transparent and unambiguous information in funding requests. 
+This template is not intended to be exhaustive and only serves as a guideline to clearly communicate the minimal amount of information 
+needed for the GRIN community to assess requests for funding. Feel free to add and modify to this template to improve clarity and transparency. 
+However, let us aim to keep this funding template minimal and elegant like GRIN and the Mimble Wimble protocol itself. 
+For ease of comparison of funding requests, it is advised to provide these fields of information in the order given below.
+
+## Financial information
+�	Funding period: 
+o	(e.g. 3 months)
+�	Total amount of requested funding:
+o	State total amount of requested funding in this period as well as in the preceding funding period both in USD and local currency
+�	Requested funding per month and number FTE�s
+o	E.g. 1 FTE is equivalent to 40 hours per week
+o	Provide requested funding per month in USDs (based on 3 months average in Google) as well as in home currency
+o	Provide the % increase/decrease in hour rate compared to previous period based on the USD value. If increased, provide an explanation why the increase is requested, e.g. I do not ask for a raise since the value requested in my home currency is the same.
+�	Country from which you work:

```

Comment:
I would simplify this section into "Country of residence", and keep this optional.